src/scripts/aerospike-client-c.sh

```diff
@@ -113,7 +113,7 @@ if [ $DOWNLOAD ] && [ $DOWNLOAD == 1 ]; then
     # LINUX
     ############################################################################
     "linux" )
-      PKG_DIST=$($SCRIPT_DIR/os_version)
+      PKG_DIST=$($SCRIPT_DIR/os_version.sh)
```

Comment:
What do you think about:
```sh
PKG_DIST=$(bash $SCRIPT_DIR/os_version)
```

instead of renaming os_version to os_version.sh?

Reply:
That would work if:
```
find vendor/aerospike/aerospike-client-php/ -name "*.sh" -exec chmod +x {} \;
```
Could find  os_version, but when it tries to execute it in some cases it fails with a permision denied. Other workarround would be to commit the file with execution permissions